lib/convert.js

```diff
@@ -121,6 +107,15 @@ Converter = {
         'persistent folder-level data.',
         external: true
       },
+      {
```

Comment:
To avoid making changes on both repositories, can we directly use the openapi.getOptions() and do merge if with any unique options present in swagger? 

In that case we won't have to do this ever again and just ensure that the recent openapi module is being used. 
@VShingala

Reply:
Yeah, it can be done but will have to avoid options for validation somehow.
Also when I did `npm i` for swagger, it did not update openapi-to-postmanv2 to latest and I had to manually bump up. AFAIK this is due to package-lock.json having 1.1.7 as version in there so we will have to keep updating package-lock.json. Thoughts?

Reply:
Yes we can remove the validation options.
Oh, that's weird. In postman-app and importer service packages I checked last it was using the latest version. Can you check in app once? Also try by using npm clean/ removing node modules.

Reply:
Tried is again on `master` but for me `npm i` is installing v1.1.7 and package-lock files remain same, ideally it should update and have latest openapi module version.

Reply:
For options, as some option contains reference link in description to openapi v3 docs, it will be confusing for someone using swagger2 and seeing them. So decided to not do it.